packages/noya-renderer/src/components/PseudoPathLine.tsx

```diff
@@ -1,25 +1,50 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { CurvePoint } from '@sketch-hq/sketch-file-format-ts/dist/cjs/types';
 import { Point } from 'noya-geometry';
 import { useStroke } from 'noya-react-canvaskit';
 import React, { useMemo } from 'react';
 import { useTheme } from 'styled-components';
-import { Polyline, Primitives } from '..';
+import { Path, Primitives, useCanvasKit } from '..';
+import { DecodedCurvePoint, encodeCurvePoint } from '../primitives/path';
 
 interface EditablePathPointProps {
   point: Point;
-  decodedCurvePoint: Primitives.DecodedCurvePoint;
+  curvePoint: CurvePoint;
+  frame: Sketch.Rect;
 }
 
 export default function PseudoPathLine({
   point,
-  decodedCurvePoint,
+  curvePoint,
+  frame,
 }: EditablePathPointProps) {
+  const CanvasKit = useCanvasKit();
   const { primary } = useTheme().colors;
-
   const stroke = useStroke({ color: primary });
 
+  const decodedPointToDraw: DecodedCurvePoint = {
+    _class: 'curvePoint',
+    cornerRadius: 0,
+    curveFrom: point,
+    curveTo: point,
+    hasCurveFrom: false,
+    hasCurveTo: false,
+    curveMode: Sketch.CurveMode.Straight,
+    point,
+  };
+
+  const encodedPointToDraw = encodeCurvePoint(decodedPointToDraw, frame);
+
   const points = useMemo(() => {
-    return [point, decodedCurvePoint.point];
-  }, [decodedCurvePoint.point, point]);
+    return [encodedPointToDraw, curvePoint];
+  }, [encodedPointToDraw, curvePoint]);
+
+  const path = Primitives.path(CanvasKit, points, frame, false);
+
+  const strokedPath = useMemo(
+    () => Primitives.getStrokedBorderPath(CanvasKit, path, 1, 0),
```

Comment:
I've hardcoded the values for the borderPath with and position, is there a use case where these values would need to be different?